docs/getting-started/quickstart.react.mdx

```diff
@@ -43,27 +43,27 @@ This tutorial will demonstrate how to create a new React app using Vite and add
     ```
   </CodeBlockTabs>
 
-  ## Install `@clerk/clerk-react`
+  ## Install `@clerk/react`
 
   The [Clerk React SDK](/docs/reference/react/overview) gives you access to prebuilt components, hooks, and helpers to make user authentication easier.
 
   Run the following command to install the SDK:
 
   <CodeBlockTabs options={["npm", "yarn", "pnpm", "bun"]}>
     ```bash {{ filename: 'terminal' }}
-    npm install @clerk/clerk-react
+    npm install @clerk/react
     ```
 
     ```bash {{ filename: 'terminal' }}
-    yarn add @clerk/clerk-react
+    yarn add @clerk/react
     ```
 
     ```bash {{ filename: 'terminal' }}
-    pnpm add @clerk/clerk-react
+    pnpm add @clerk/react
     ```
 
     ```bash {{ filename: 'terminal' }}
-    bun add @clerk/clerk-react
+    bun add @clerk/react
     ```
   </CodeBlockTabs>
```

Comment:
You can replace all of `CodeBlockTabs` with the following and it will automatically create yarn, pnpm, and bun commands.

~~~
```npm
npm install @clerk/react
```
~~~